addons/account_qr_code_emv/models/res_bank.py

```diff
@@ -23,25 +25,35 @@ def _get_crc16(self, data, poly=0x1021, init=0xFFFF):
     def _get_merchant_account_info(self):
         return False
 
+    def _get_additional_data_field(self, comment):
+        return False
+
     def _get_qr_vals(self, qr_method, amount, currency, debtor_partner, free_communication, structured_communication):
         if qr_method == 'emv_qr':
             merchant_account_info = self._get_merchant_account_info()
             currency_code = CURRENCY_MAPPING[currency.name]
             merchant_name = self.partner_id.name and self.partner_id.name[:25] or 'NA'
             merchant_city = self.partner_id.city and self.partner_id.city[:15] or ''
+            comment = (free_communication or '') if not structured_communication else ''
+            comment = re.sub(r'[^A-Za-z0-9]+', '', comment)
```

Comment:
:thinking: I wonder if removing spaces is a good idea. Free communications often contain texts no? In which case it would be rendered unusable.
Structured communications also often contain "/" or "+" 
The field is marked as "Alphanumeric Special" on the doc, can't we keep some of these?